src/main/java/seedu/address/logic/parser/RecipeBookParser.java

```diff
@@ -56,8 +56,8 @@ public Command parseCommand(String userInput) throws ParseException {
             case AddCommand.COMMAND_WORD:
                 return new AddCommandParser().parse(arguments);
             case ClearCommand.COMMAND_WORD:
-                requiresConfirmationClear = ClearCommandParser.isNotForcedClear(arguments);
-                return new ClearCommandParser().parse(arguments);
+                requiresConfirmationClear = ClearCommandParser.isNotForcedClear(userInput);
```

Comment:
Was there a reason to why this was changed to userInput?

Reply:
Was this explained? 👀